core/templates/components/skill-selector/select-skill-modal.component.ts

```diff
@@ -41,7 +45,8 @@ export class SelectSkillModalComponent extends ConfirmOrCancelModal {
   }
 
   confirm(): void {
-    let totalSkills = [];
+    let totalSkills:
+    (ShortSkillSummary | SkillSummaryBackendDict)[] = [];
```

Comment:
Ditto as above, also should we define common type?

Reply:
@vojtechjelinek please checkout the changes I've made. 
`totalSkills` is now an array of one type `SkillSummaryBackendDict` since it is the the interface of `SkillSummary` with all properties

Converted `ShortSkillSummary` to `SkillSummaryBackendDict` with the use of `as` as done in typescript.